src/main/groovy/de/aaschmid/gradle/plugins/cpd/Cpd.groovy

```diff
@@ -215,7 +214,18 @@ class Cpd extends SourceTask implements VerificationTask, Reporting<CpdReports>
         }
     }
 
-    /**
+    private String asClickableFileUrl(File path) {
+        // File.toURI().toString() leads to an URL like this on Mac: file:/reports/index.html
+        // This URL is not recognized by the Mac console (too few leading slashes). We solve
+        // this be creating an URI with an empty authority.
+        try {
+            return new URI("file", "", path.toURI().getPath(), null, null).toString();
+        } catch (URISyntaxException e) {
+            throw UncheckedException.throwAsUncheckedException(e);
```

Comment:
You need to copy [UncheckedException](https://github.com/gradle/gradle/blob/f490bdf61bd9b4f5383cd9fb0d8ffbca93da8c32/subprojects/base-services/src/main/java/org/gradle/internal/UncheckedException.java) as well.